app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,26 @@ def show
     @posts = @user_post.posts
     @post = @posts.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @post } }
+    end
+  end
+
+  def create
+    @post = current_user.posts.new(params.require(:post).permit(:Title, :Text))
```

Comment:
- [Optional] I suggest you to use strong parameters. Check [this link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to learn more: